src/olympia/files/admin.py

```diff
@@ -20,7 +20,13 @@ def has_add_permission(self, request, obj=None):
 
 class WebextPermissionInline(admin.StackedInline):
     model = WebextPermission
-    fields = ('permissions', 'optional_permissions', 'host_permissions')
+    fields = (
+        'permissions',
+        'optional_permissions',
+        'host_permissions',
```

Comment:
Unrelated to this PR but we also have `optional_host_permissions`: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/optional_host_permissions